README.md

```diff
@@ -1,113 +1,101 @@
-# The PrimeDAO’s Home for IDOs
+# Prime Deals dApp
+Prime Deals is an An Interface for DAO to DAO interactions, such as token swaps, co-liquidity provision, and joint venture formation. An introduction to Prime Deals [TODO: ADD LINK](link-to-introduction).
 
-This is the [PrimeDAO](primedao.eth.link) web interface enabling the creation of and interaction with IDOs.
+This prime-deals-dapp repository contains in its master branch the web user interface for the Prime Deals website that is deployed to https://deals.prime.xyz.
 
-## Technical Description
+## Development
 
-This project is bootstrapped by [aurelia-cli](https://github.com/aurelia/cli).
+### Prerequisites
+Make sure you have [node.js version >= 14.11.0 < 15](https://nodejs.org/en/)
 
-It is written mostly in Typescript, HTML and SCSS, and is bundled using Webpack.
-
-For more information about bundling, go to https://aurelia.io/docs/cli/webpack
-
-## Install
+### Install
 Install dependencies with the following command:
 ```
 npm ci
 ```
+### Update Contract ABIs
+Prime Deals relies on solidity contract addresses and ABIs that it obtains from the [TODO: ADD LINK TO CONTRACTS REPO](https://github.com/PrimeDAO/). You must clone the contracts repository in a folder sibling to this one.
 
-## <a name="build"></a> Build
-There are several ways to build and run the application.  Each of the following build commands will output to the `dist` folder. After building, run following to launch the result in the browser:
-
+Then run the following script to pull the required contract ABIs from contracts:
 ```
-npm run start
+npm run fetchContracts
 ```
+You only need run this script once, or else again when any of the contracts change.
 
-### Build with optimized code against mainnet
-`npm run build`
-
-(This is the production build.)
-
-### Build with optimized code against kovan
-
-`npm run build-kovan`
-
-### Build unoptimized code against kovan
-
-`npm run build-dev`
-
-### Build unoptimized code against mainnet
-
-`npm run build-dev-mainnet`
+### Build
+The package.json file contains lots of commands for building or serving up the application.
 
-## Debug
+#### Environment
+Before building, make sure to have the following in your OS environment variables or in an ".env" file:
+```jsonc
+RIVET_ID=...
+INFURA_ID=...
+ETHPLORER_KEY=...
 
-Each of the following commands will serve up a site that will support Hot Module Reload (HMR).  You can then use your favorate debugger to launch the app at http://localhost:3340.
+PINATA_API_KEY_TEST=...
+PINATA_SECRET_API_KEY_TEST=...
 
-### Build and serve, running against kovan
-`npm run serve-dev`
+IPFS_GATEWAY="https://${hash}.${protocol}.dweb.link/"
 
-### Build and serve, running against mainnet
-`npm run serve-dev-mainnet`
+NETWORK=rinkeby // or mainnet
```

Comment:
I wouldn't include NETWORK and PORT here because they aren't concerned about vercel

Reply:
Should include "COINGECKO_API_KEY="

Reply:
> I wouldn't include NETWORK and PORT here because they aren't concerned about vercel

I disagree regarding the NETWORK variable. When someone would try to deploy the app, NETWORK is required, either in the platform environment variables definitions or the run command. But it needs to be clear that the variable needs to be passed.

Reply:
Thing is, the variable does not "need" to be passed.  Note in main.ts it defaults to rinkeby or mainnet, depending on whether is a development or production build.  So really, the only concern here is Vercel, and that is only *our* concern. We are not concerned here about how other people might deploy the app to their own idiosyncratic servers.  I still think should leave NETWORK out.

Reply:
Oh, fair point. Didn't see that defaulting in main.js.